packages/devtools_app/lib/src/screens/debugger/span_parser.dart

```diff
@@ -650,3 +542,201 @@ class _IncludeMatcher extends _Matcher {
     };
   }
 }
+
+/// Tracks the current scope stack, producing [ScopeSpan]s as the contents
+/// change.
+class ScopeStack {
+  ScopeStack();
+
+  final Queue<ScopeStackItem> stack = Queue();
```

Comment:
Nits: drop the types on the LHS and use explicit types on the RHS instead for `stack` and `spans`.

Reply:
Done!